template/template.go

```diff
@@ -38,21 +55,27 @@ func Manga(_ string, manga mangadata.Manga) string {
 	err := template.Must(template.New("manga").
 		Funcs(funcs.FuncMap).
 		Parse(plt)).
-		Execute(&sb, manga)
+		Execute(&sb, mangaTemplateData{
+			Manga:    manga.Info(),
+			Metadata: manga.Metadata(),
+		})
```

Comment:
The case for `manga.Metadata() == nil` should be checked (alongside where the `plt` is assigned) and if `nil` then pass `&metadata.Metadata{}` to avoid panics.